package.json

```diff
@@ -66,10 +72,11 @@
   "scripts": {
     "dev": "webpack-dev-server --content-base WebContent --inline --hot",
     "start": "npm run dev",
-    "lint": "eslint ./WebContent/js/ ./tests/",
+    "lint": "eslint ./WebContent/js/ ./tests/UnitTests",
     "testWatch": "mocha --watch --require babel-core/register tests --recursive",
-    "test": "JUNIT_REPORT_PATH=target/report.xml nyc mocha --require babel-core/register tests --recursive --colors --reporter mocha-jenkins-reporter",
+    "test": "JUNIT_REPORT_PATH=target/report.xml nyc mocha --require babel-core/register tests/UnitTests --recursive --colors --reporter mocha-jenkins-reporter",
     "prod": "cross-env NODE_ENV='production' webpack && cp -r ./WebContent/css ./dist/ && cp -r ./WebContent/img ./dist/ && cp ./WebContent/index.html ./dist/ && cp ./WebContent/favicon.ico ./dist/",
+    "test:fvt": "JUNIT_REPORT_PATH=target/report-fvt.xml mocha --timeout 999999 --colors tests/FVTTests --recursive  --reporter mocha-jenkins-reporter",
```

Comment:
suffix with cross-env for windows user

Reply:
Have added cross-env to the test commands